src/components/ResortsData.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import ResortsList from './ResortsList';
+import ResortsListHead from './ResortsListHead';
+import Pagination from './Pagination';
+
+class ResortsData extends Component{
+  constructor(){
+    super();
+    this.state = {
+      data: [],
+      page: 3,
+      isLoading: false,
+      error: null
+    }
+  }
+  
+  fetchResorts(){
+    this.setState({isLoading: true})
+    fetch(`http://localhost:3000/resorts?_page=${this.state.page}&_limit=1`)
+        .then(response => {
+          if (response.ok) {
+            return response.json();
+          } else {
+            throw new Error('Something went wrong ...');
+          }
+        })
+        .then(data => this.setState({ data: data, isLoading: false }))
+        .catch(error => this.setState({ error, isLoading: false}))
+  }
+
+  componentDidMount(){
+    this.fetchResorts();
+  }
+
+  handleClick(page) {
```

Comment:
If you define this function as fat arrow `handleClick = (page) => {` you will not have to use `bind(this)` later.